pg/benchmarks/pg_scope_all.rb

```diff
@@ -0,0 +1,23 @@
+require 'bundler/setup'
+require 'pg'
+require_relative 'support/benchmark_pg'
+
+# Connect to POSTGRES database with provided values or use default
+conn = PG.connect(host: ENV.fetch("HOST", "localhost"),
```

Comment:
I prefer

```
conn = PG.connect(
  host: ENV.fetch("HOST", "localhost"),
  port: ..
)
```

The problem I have with flushing indentations is that changing anything in the first line forces us to re-indent all the lines polluting the commit history.